pages/account/statistics.js

```diff
@@ -77,7 +79,9 @@ export default function Search({ data, profile }) {
   const dateTimeStyle = {
     dateStyle: "short",
   };
-  const dailyViews = data.profile.daily.slice(-30).map((day) => {
+  
+  let sortedByDate = data.profile.daily.sort((a, b) => new Date(a.date) - new Date(b.date));
```

Comment:
I think we can sort on the server side [here](https://github.com/EddieHubCommunity/LinkFree/blob/auth-stats/pages/api/account/statistics.js#L36) (but I sorted by views instead of date, I think I did a copy and paste and forgot to update it 🤦‍♂️ )

Reply:
Thank you for noticing my mistake 👍 , I just pushed a fix